packages/kas/src/nodes.ts

```diff
@@ -310,12 +347,12 @@ class Expr {
     }
 
     // whether this node needs an explicit multiplication sign if following a Num
-    needsExplicitMul() {
-        return this.args()[0].needsExplicitMul();
+    needsExplicitMul(): boolean {
+        return this.exprArgs()[0].needsExplicitMul();
```

Comment:
Is this essentially the same, just more explicit?

Reply:
Without it TS thinks the return type is `any`.  Sometimes TS type inference is just really bad.  I tried turning on `noImplicitAny: true` in the tsconfig.json but then it was complaining about things that it was able to infer as a real type. 😞